code/modules/admin/admin_verbs.dm

```diff
@@ -1017,16 +1016,3 @@ var/list/admin_verbs_ticket = list(
 
 	log_admin("[key_name(usr)] has [advanced_admin_interaction ? "activated" : "deactivated"] their advanced admin interaction.")
 	message_admins("[key_name_admin(usr)] has [advanced_admin_interaction ? "activated" : "deactivated"] their advanced admin interaction.")
-
-/client/proc/toggle_panic_bunker()

```

Comment:
Don't remove this, add a flag for "manual override" or something

Reply:
I'm actually fine with removing this, and having the system be entirely automated. It isn't a requirement, but this really isn't a system that needs manual control. Server capacity doesn't change round to round.

Reply:
I have added a new admin verb `Set Server Capacity` that prompts the admin for a number.
The configuration variable `config.panic_bunker_threshold` will be set to the value provided.
It's not the same as a toggle on/off, but it allows admin control just the same.

Reply:
I don't want a Set Server Capacity verb for admins.
I'm okay with a literal on/off switch for admins, so long as it affects only the remainder of the current round.
I'm not okay with any admin being able to change this in a way that persists beyond the current round.
I'm not okay with any admin being able to set this to say, 50, or 200. That just seems like it would create a bunch of problems for me to deal with.